packages/opc-header/dist/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>opc-header</title>
+    <script src="opc-header.js"></script>
+    <style>
+        body {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            height: 100vh;
+        }
+        .test__styles {
+            font-size: 2rem;
+        }
+    </style>
+</head>
+<body>
+    <opc-header style="--opc-header--width: 1200px;">
+        <div slot="breadcrumb-slot">
+            This is a breadcrumb
+        </div>
+        <div slot="right-link-slot" class="test__styles">
+            Hello World
+        </div>
+    </opc-header>
+</body>
```

Comment:
Can you following example... 
with Default theme: 
1. Only Header
2. With Header and Breadcrumb
3. With Header and Breadcrumb and side links

Reply:
you mean i need to use it <opc-header> 3 times ?.

Reply:
Updated changes. please have a look.